### PR TITLE
Various minor UI tweaks

### PR DESCRIPTION
**Pull request by lintool**
Changed order of fields in metadata record.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -224,8 +224,11 @@ function parseMetadata(dataset){
...
     for (let i = 0; i < responsibleParties.length; i++) {
-        responsiblePartiesString += responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:individualName"] + ", " + 
-                                    responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:role"]["gmd:CI_RoleCode"] + "<br/>"
+        role = responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:role"]["gmd:CI_RoleCode"];
+        if ( role === 'principalInvestigator' )
+            role = 'Principal Investigator';
+        responsiblePartiesString += responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:individualName"] + " (" + 
+                                    role + ")<br/>";
     }
     $('#meta-responsibleParties').html(responsiblePartiesString.substring(0, responsiblePartiesString.length - 1));
```
Comment (gauchm) [inline index.js]: Not your mistake, but this should be `length - 5` to remove the whole `<br/>`.
```suggestion
     $('#meta-responsibleParties').html(responsiblePartiesString.substring(0, responsiblePartiesString.length - 5));
```

File: index.js [hunk 2]
```diff
@@ -224,8 +224,11 @@ function parseMetadata(dataset){
     $('#meta-title').html(title);
     var responsiblePartiesString = "";
     for (let i = 0; i < responsibleParties.length; i++) {
-        responsiblePartiesString += responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:individualName"] + ", " + 
-                                    responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:role"]["gmd:CI_RoleCode"] + "<br/>"
+        role = responsibleParties[i]["gmd:CI_ResponsibleParty"]["gmd:role"]["gmd:CI_RoleCode"];
+        if ( role === 'principalInvestigator' )
```
Comment (gauchm) [inline index.js]: Role also appears in line 218. So I'd suggest to add a method `formatRole` that does the formatting.
Plus, there are some more unreadable role texts (`pointOfContact` and lowercase `originator`/`distributor`), so the method would be something like:
```
function formatRole(role) {
    if (role === 'principalInvestor') {
        role = 'Principal Investor';
    } else if (role === 'pointOfContact') { 
        role = 'Point of Contact';
    }
    return role.charAt(0).toUpperCase() + string.slice(1);
}
```

**Comment by lintool** [comment]
Addressed CR!

**Review by gauchm — APPROVED** [review]
Looks good!

**Pull request merged**